nums/numpy/api.py

```diff
@@ -382,10 +382,22 @@ def where(condition, x=None, y=None):
     return _instance().where(condition)
 
 
-def all(a: BlockArray, axis=None, out=None, keepdims=False):
+def all(a: BlockArray, axis=None, out=None, keepdims=False, dtype=None):
```

Comment:
Can we remove `dtype=None`? It's not a part of the NumPy API (If someone goes from using NumS to NumPy, their code ought to work).

Reply:
Yep, will do.